rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/InitScriptGenerator.java

```diff
@@ -35,55 +35,84 @@
 
 import com.google.common.base.Joiner;
 
+import lombok.Getter;
+
 
 public class InitScriptGenerator {
 
-    public static final String DETACHED_MODE_PREFIX_PROPERTY = "%detachedModePrefix%";
+    private static final String DETACHED_MODE_PREFIX_PROPERTY = "%detachedModePrefix%";
+
+    private static final String JAVA_PATH_PROPERTY = "%javaPath%";
+
+    private static final String SCHEDULING_PATH_PROPERTY = "%schedulingPath%";
+
+    private static final String FILE_ENCODING = "%fileEncoding%";
+
+    private static final String JAVA_OPTIONS_PROPERTY = "%javaOptions%";
+
+    private static final String RM_URL_PROPERTY = "%rmUrl%";
+
+    private static final String NODE_NAME_PROPERTY = "%nodeName%";
+
+    private static final String NODE_SOURCE_NAME_PROPERTY = "%nodeSourceName%";
 
-    public static final String JAVA_PATH_PROPERTY = "%javaPath%";
+    private static final String CREDENTIALS_PROPERTY = "%credentials%";
 
-    public static final String SCHEDULING_PATH_PROPERTY = "%schedulingPath%";
+    private static final String NUMBER_OF_NODES_PER_INSTANCE_PROPERTY = "%numberOfNodesPerInstance%";
 
-    public static final String FILE_ENCODING = "%fileEncoding%";
+    private static final String JAVA_URL_PROPERTY = "%javaUrl%";
 
-    public static final String JAVA_OPTIONS_PROPERTY = "%javaOptions%";
+    private static final String NODE_JAR_URL_PROPERTY = "%nodeJarUrl%";
 
-    public static final String RM_URL_PROPERTY = "%rmUrl%";
+    private static final String JYTHON_PATH_PROPERTY = "%jythonPath%";
 
-    public static final String NODE_NAME_PROPERTY = "%nodeName%";
+    private static final String NODE_JAR_URL_PROTOCOL_PROPERTY = "%protocol%";
 
-    public static final String NODE_SOURCE_NAME_PROPERTY = "%nodeSourceName%";
+    private static final String NODE_JAR_URL_HOST_NAME_PROPERTY = "%hostName%";
 
-    public static final String CREDENTIALS_PROPERTY = "%credentials%";
+    private static final String NODE_JAR_URL_PORT_PROPERTY = "%port%";
 
-    public static final String NUMBER_OF_NODES_PER_INSTANCE_PROPERTY = "%numberOfNodesPerInstance%";
+    private static final String NODE_JAR_URL_DIR_PROPERTY = "%nodeJarDir%";
+
+    private static final String NODE_JAR_DIR = "rest/node.jar";
```

Comment:
NODE_JAR_DIR -> NODE_JAR_URL_PATH ? (DIR seems more to be the disk location on machine for me)